sdk/storage/azure-storage-blob/azure/storage/blob/_policies.py

```diff
@@ -0,0 +1,39 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint:disable=line-too-long
+from typing import Any, TYPE_CHECKING
+
+from azure.core.pipeline.policies import (
+    SansIOHTTPPolicy
+)
+from ._shared.request_handlers import throw_exception_if_contains_header
+from ._generated.version import VERSION
+
+if TYPE_CHECKING:
+    from azure.core.pipeline import PipelineRequest
+
+Managed_DISK_DIFF_OPERATION = 'get_managed_disk_page_range_diff'
+CREATE_CONTAINER_OPERATION = 'create container'
+
+
+class StorageVersionCheckPolicy(SansIOHTTPPolicy):
```

Comment:
Should this include the prefix Blob? Not sure what the Python guidelines are for naming

Reply:
create_container is the API name, so I didn't put a prefix here. What do you think?